### ISSUE TITLE
0.11.0 published without dist/tokenizer directory

### ISSUE THREAD

**Issue by spalger** [webschik/postcss-less#40]
It looks like the tarball distributed for 0.11.0 is invalid.

![image](https://cloud.githubusercontent.com/assets/1329312/15288268/235de758-1b1e-11e6-8b97-1c33105d478d.png)

**Comment by webschik** [comment]
Hi @spalger! 
Are you about '.min.js' files?

**Comment by spalger** [comment]
No, when I require this plugin in my webpack config I get this stack trace:

```
Module build failed: Error: Cannot find module './tokenizer/globals'
    at Function.Module._resolveFilename (module.js:438:15)
    at Function.Module._load (module.js:386:25)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/spalger/dev/cloud/cloud-adminconsole-frontend/node_modules/postcss-less/dist/is-mixin-token.js:8:16)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/spalger/dev/cloud/cloud-adminconsole-frontend/node_modules/postcss-less/dist/less-parser.js:21:21)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
```

**Comment by webschik** [comment]
@spalger, @paazmaya fixed it in v0.12.0
